server/views/pages/overview/partials/supportNeedsTabContents.test.ts

```diff
@@ -27,17 +27,41 @@ describe('Support Needs tab view', () => {
     const $ = cheerio.load(compiledTemplate.render(viewContext))
 
     // Then
```

Comment:
This test has had to change quite a lot, because the markup in the nunjucks template has changed a lot.
I'm also found this syntax which I think reads quite well:
```
    expect(
      healthAndSupportNeedsCard //
        .find(`.govuk-summary-list__key:contains('Rapid assessment')`)
        .next()
        .text()
        .trim(),
    ).toEqual('18 February 2022')
```
IE. assert the value of the Radid Assessment dt/dd is 18 February 2022